Dockerfile.php5.6

```diff
@@ -78,4 +78,20 @@ RUN set -ex; \
 			| sort -u \
 	)"; \
 	apk add --virtual .wordpress-phpexts-rundeps $runDeps; \
-	apk del .build-deps
+	apk del .build-deps; \
+	rm -rf /tmp/pear/;
+
+RUN set -ex; \
+	\
+	NR_VERSION="$( \
+		curl -sS https://download.newrelic.com/php_agent/release/ \
```

Comment:
should use `--connect-timeout` on curl so it doesn't block forever if something goes wrong network-wise